widget/embedded/src/store/slices/data.ts

```diff
@@ -27,6 +27,7 @@ type TokenOptions = {
 export type FetchStatus = 'loading' | 'success' | 'failed';
 export interface DataSlice {
   _blockchainsMapByName: Map<string, BlockchainMeta>;
+  _tokensMap: Map<string, Token>;
```

Comment:
I think it is better to use a clearer name like `_blockchainsMapByName`.

Suggested name: _tokensMapByHashToken

Reply:
fixed